pytest/tests/sanity/split_storage.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+"""
+Spins up an archival node with cold store configured and verifies that blocks are copied from hot to cold store.
+"""
+
+import sys
+import pathlib
+import os
+import copy
+import json
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[2] / 'lib'))
+
+from utils import poll_blocks
+from cluster import init_cluster, spin_up_node, load_config, get_config_json, set_config_json
+from configured_logger import logger
```

Comment:
This totally messes up my vs code. It doesn't understand where from to import those libraries and plenty of features don't work. Has anyone looking into improving that?